syntax = "proto3";

import "google/protobuf/empty.proto";

package redis;

service RedisService {
    rpc getIp (IpRequest) returns (IpResponse) {}
    rpc setUser (UserRequest) returns (google.protobuf.Empty) {}
}

message UserRequest {
    uint32 user_id = 1;
    string ip_v4 = 2;
}

message IpRequest {
    uint32 user_id = 1;
}

message IpResponse {
    string ip_v4 = 1;
}